doc/changes/changes_5.1.3.md

```diff
@@ -0,0 +1,34 @@
+# Test containers for Exasol on Docker 5.1.3, released 2022-01-20
```

Comment:
@redcatbear is it ok to do this in a fix release or should we create a new minor / major version?

Reply:
This is a breaking change, so we must increase the major number.

Reply:
ok